src/controller/WebController.php

```diff
@@ -503,7 +489,8 @@ public function invokeVocabularyHome($request)
     {
         $lang = $request->getLang();
         // set language parameters for gettext
-        $this->setLanguageProperties($lang);
+        //$this->setLanguageProperties($lang);
```

Comment:
same as above, remove

Reply:
Removed in new commit: https://github.com/NatLibFi/Skosmos/commit/48b31a51acb2fd8be2452c033e0cecd05d86c764